components/centraldashboard/webpack.config.js

```diff
@@ -206,6 +206,10 @@ module.exports = {
                         '/$1/services/$2/proxy/notebook/$1/$2/$3',
                 },
             },
+            '/pipeline': {
+                target: 'http://localhost:8083/api/v1/namespaces/kubeflow/services/ml-pipeline:8888/proxy',
```

Comment:
What's this?

Reply:
During development, this allows requests made from the browser that are destined for URLs that would only be available in cluster (ie. `/jupyter/` or `/pipeline/`) to be proxied to a local Kubernetes proxy server that can be started by running `kubectl proxy -p 8083`.

So when you run `npm run dev`, the front-end is served on port 8081. Our API server runs on port 8082 with any request made from the front-end to `/api` being proxied there. These additional lines allow you to communicate with the other services that will be available at specific top-level routes when deployed.

Here's some more information on what you can do with `kubectl proxy`.
https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api